docs/content/docs/how-to/add-handwritten-test.md

```diff
@@ -192,6 +192,102 @@ resource "google_compute_firewall" "foobar" {
 }
 ```
 
+## Delete Tests
+
+It is necessary to add a delete test to see if a resource can be properly removed, especially
```

Comment:
I like the idea of documenting this, but it'll need to be really clear about the reason that someone might want to do this, since generally this is not necessary. CryptoKeyVersion needed this kind of test because we needed to make sure the resource was being deleted due to the CryptoKeyVersion being removed from Terraform state (because it would also be deleted if the CryptoKey was deleted)